source/common/http/utility.h

```diff
@@ -240,6 +240,14 @@ absl::string_view findQueryStringStart(const HeaderString& path);
  **/
 std::string parseCookieValue(const HeaderMap& headers, const std::string& key);
 
+/**
+ * Parse a particular value out of a set-cookie
+ * @param headers supplies the headers to get the set-cookie from.
+ * @param key the key for the particular set-cookie value to return
+ * @return std::string the parsed set-cookie value, or "" if none exists
+ **/
+std::string parseSetCookieValue(const HeaderMap& headers, const std::string& key);
```

Comment:
Mind adding a test for this function? Ideally we have coverage for this outside of extensions. Thanks!

Reply:
I can copy-past some tests from utility_test.cpp with parseCookieValue check and trasnform them for parseCookieValue. Will it be okay?